File: midterm/QueueSample.java
package midterm;

import java.util.*;

public class QueueSample {
    
    public static void main(String[] args) {
        Queue<String> foods = new  LinkedList<String>();

        // i add food to the queue

        foods.offer("Tapsilog");
        foods.offer("BBQ");
        foods.offer("Siomai Rice");
        foods.offer("Sisig");
        foods.offer("Burger Steak");

       
        System.out.println("Available foods: " + foods);
        System.out.println("The first food to be served is: "+ foods.peek());

         // i remove food to the queue until it becomes empty!
        while(!foods.isEmpty()) {
            System.out.println("The food "+ foods.poll() + " has been removed to the queue");
            
            
        }
        System.out.println("Available foods: "+ foods);

        if(foods.isEmpty()) {
            System.out.println("No more foods!!!!");
        }else {
            System.out.println("There is more foods!");
        }
    }

}

File: ds-algo/midterm/queue_sample.py
from collections import deque

class QueueSample:

    foods = deque()

    # I add food to the queue
    foods.append("Tapsilog")
    foods.append("BBQ")
    foods.append("Siomai Rice")
    foods.append("Sisig")
    foods.append("Burger Steak")

    # I display the available foods
    print("Available foods: " + str(foods))

    # I check the first food in the queue
    print("The first food to be served is: " + str(foods[0]))

    # I will remove until the queue is empty

    while foods:
        print("The food " + str(foods.popleft()) + " has been removed frin the queue.")
    
    # I display the list to check if the queue is empty 
    print("Available foods: " + str(foods))

    if not foods:
        print("No more foods!!!")
    else:
        print("There is more foods!")


    